raspbian/applications/camera.md

```diff
@@ -368,8 +368,8 @@ Text may include date/time placeholders by using the '%' character, as used by <
 |-a 512|Frame Number| |
 |-a 1024|Black Background| |
 |-a "ABC %Y-%m-%d %X"|Show some text|ABC %Y-%m-%d %X|
-|-a 4 -a "ABC %Y-%m-%d %X"|Show custom <a title="strftime man page" href="http://man7.org/linux/man-pages/man3/strftime.3.html">formatted</a> date/time|ABC 2015-10-28 20:09:33|
-|-a 8 -a "ABC %Y-%m-%d %X"|Show custom <a title="strftime man page" href="http://man7.org/linux/man-pages/man3/strftime.3.html">formatted</a> date/time|ABC 2015-10-28 20:09:33|
+|-a 4 -a "ABC %Y-%m-%d %X"|Show custom <a title="strftime man page" href="https://www.man7.org/linux/man-pages/man3/strftime.3.html">formatted</a> date/time|ABC 2015-10-28 20:09:33|
```

Comment:
Can anyone think of a reason why these HTML-style links shouldn't be converted to MarkDown-style links?

Reply:
No. I don't see why any reason they have to be HTML-style.

Reply:
The tooltip text will be missing with the Markdown syntax.

Reply:
I wonder why I used HTML there and not markdown? Or maybe it was a third party edit after the original was written. Hmm. Anyway, I woudl have thought markdown was sufficient.